include/exadg/convection_diffusion/user_interface/application_base.h

```diff
@@ -92,15 +91,15 @@ class ApplicationBase
     param.check();
     param.print(pcout, "List of parameters:");
 
-    // grid
+    // create grid and mapping owned by driver
```

Comment:
just as a minor comment: I think I would not introduce here in the comment the notion `driver` and where this variable comes from. Considering ExaDG as a library, one may use the `ConvDiff::Application` and `Grid` classes in ExaDG or derived projects independently of `ConvDiff::Driver`.